llm/llm_functions.py

```diff
@@ -532,6 +653,58 @@ def get_tags(bill_details: BillDetails) -> tuple[int, LLMResults]:
 
     return 1, tag_response
 
+def get_tags_v2(bill_details: BillDetails) -> LLMResults:
```

Comment:
Can we call this `get_tags_by_summaries`? And then rename the v1 to `get_tags_by_bill_text`?